srcpkgs/pipewire/files/pipewire/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+! [ -d /run/pipewire ] && install -m 755 -g pipewire -o pipewire -d /run/pipewire
+umask 002
+PIPEWIRE_RUNTIME_DIR=/run/pipewire XDG_CONFIG_HOME=/var/lib exec chpst -u pipewire:pipewire pipewire
```

Comment:
needs pipewire:pipewire:**audio:video** too to work

Reply:
Could you also split this line into multiple lines with `export ...` to make it more readable? same in pipewire-pulse

Reply:
> needs pipewire:pipewire:**audio:video** too to have the permissions to open hardware devices

done, this is not needed in the pipewire-pulse service right?

Reply:
> done, this is not needed in the pipewire-pulse service right?

right